picard/ui/theme.py

```diff
@@ -33,6 +36,57 @@
 )
 
 
+OS_SUPPORTS_THEMES = True
+if IS_MACOS:
+    def is_dark_theme_supported():
+        import platform
+        try:
+            current_version = tuple(map(int, platform.mac_ver()[0].split(".")))[:2]
+        except ValueError:
+            log.warning("Error while converting the MacOS version string into a tuple: %s" % platform.mac_ver()[0])
+            return False
+
+        mojave_version = (10, 14)  # Dark theme support was introduced in Mojave
+        return current_version >= mojave_version
+    OS_SUPPORTS_THEMES = is_dark_theme_supported()
+    del is_dark_theme_supported
```

Comment:
Not sure it is worth deleting the function, that's quite unusual, and prolly cost more than not doing it.

Reply:
It looks like an unnecessary and in the end pointless small optimization to me. I personally would not do this. It's no big deal either way, but it won't have any real effect, neither positively nor negatively.

Reply:
@Gabrielcarvfer I guess you can remove this `del`